src/sshRunner.ts

```diff
@@ -99,48 +111,78 @@ export class SSHRunner extends TerminalBaseRunner {
             }
         }
 
-        openSSHConsole(this._outputChannel, targetServer)
-            .then((terminal) => {
-                if (!terminal) {
-                    this._outputChannel.appendLine('\nSSH connection failed.');
-                    this._outputChannel.show();
-                    return;
+        let terminal = undefined;
+
+        let reuse = utilities.getCodeConfiguration<boolean>('ansible', 'reuseSSHTerminal');
+
+        if (reuse) {
+            let terminalNames = Object.keys(this.terminalList);
+            for (let t of terminalNames) {
+                if (t === this.getTerminalName(targetServer.host)) {
+                    terminal = this.terminalList[t];
+                    break;
                 }
-                var count: number = 60;
-                var _localthis = this;
-                var connected = false;
-
-                const tempFile = path.join(os.tmpdir(), 'vscodeansible-ssh-' + targetServer.host + '.log');
-
-                var interval = setInterval(function () {
-                    count--;
-                    if (count > 0) {
-                        if (fs.existsSync(tempFile)) {
-                            count = 0;
-                            fs.removeSync(tempFile);
-                            connected = true;
-
-                            if (utilities.isTelemetryEnabled()) {
-                                terminal.sendText('export ' + Constants.UserAgentName + '=' + utilities.getUserAgent());
+            }
+        }
+
+        if (terminal) {
+            terminal.show();
+
+            cmds.push('ansible-playbook ' + targetPlaybook);
```

Comment:
How will this work with other PR which introduced ansible-playbook params?

Reply:
merge. so  pls review them, then i can do merge